etna/models/deadline_ma.py

```diff
@@ -40,7 +47,8 @@ def __init__(self, window: int = 3, seasonality: str = "month"):
             Only allowed monthly or annual seasonality.
         """
         self.window = window
-        self.seasonality = SeasonalityMode(seasonality)
+        self.seasonality = seasonality
+        self._seasonality = SeasonalityMode(seasonality)
```

Comment:
This is done because `to_dict` works incorrectly with enums. We should probably fix this somehow, but here I made a simple fix.

Reply:
May you add issue about it and TODO here